core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/convert.kt

```diff
@@ -49,12 +55,171 @@ import kotlin.reflect.KType
 import kotlin.reflect.full.isSubtypeOf
 import kotlin.reflect.typeOf
 
+/**
+ * See also [parse] — a specialized form of the [convert] operation that parses [String] columns
+ * into other types without requiring explicit type specification.
+ */
+internal interface SeeAlsoParse
+
+/**
+ * Converts the values in the specified [columns\] either to a supported target type
+ * or using a custom converter, keeping their original names and positions within the [DataFrame].
+ *
+ * This function does not immediately convert the columns but instead selects columns to convert and
+ * returns a [Convert],
+ * which serves as an intermediate step.
+ * The [Convert] object provides methods to transform selected columns using:
+ * - [to][Convert.to]
+ * - [with][Convert.with]
+ * - [asFrame][Convert.asFrame]
+ * - [perRowCol][Convert.perRowCol]
+ * - [notNull][Convert.notNull]
+ * - [toDataFrames][Convert.toDataFrames]
+ *
+ * Additionally, it offers a wide range of methods for converting to specific types,
+ * such as [toStr][Convert.toStr], [toDouble][Convert.toDouble], and many others.
+ *
+ * For the full list of supported types,
+ * refer to the [documentation website]({@include [DocumentationUrls.Url]}/convert.html).
```

Comment:
please add this urls in the file instead of here in the doc. Makes it easier to refactor if all urls are in 1 place

Reply:
in all places you use this btw